tests/integration/test_scheduler.py

```diff
@@ -346,6 +346,17 @@ async def test_restart_timeout(
 
     See: https://github.com/cylc/cylc-flow/issues/5078
     """
+    one_conf.update(
+        {
+            'runtime': {
+                'one': {
+                    'simulation': {
+                        'default run length': 'PT5S'
```

Comment:
I've got an auxiliary PR to simplify this back down: https://github.com/hjoliver/cylc-flow/pull/60